### PR TITLE
md5: split basic example for simplicity, add example of md5 checksum for given string

### PR DESCRIPTION
**Pull request by igorshubovych**

### CODE REVIEW

File: pages/osx/md5.md [hunk 1]
```diff
@@ -2,12 +2,19 @@
 
 > Calculate MD5 cryptographic checksums
 
-- Calculate the MD5 checksum for file(s) or files in a directory, one checksum per file
+- Print MD5 checksum for file
```
Comment (waldyrious) [inline pages/osx/md5.md]: I'd suggest "Calculate the MD5 checksums", especially since you're adding an example that isn't a file.

**Comment by waldyrious** [comment]
:+1: LGTM :)

File: pages/osx/md5.md [hunk 2]
```diff
@@ -2,12 +2,19 @@
...
 
 `md5 {{filename1}}`
```
Comment (waldyrious) [inline pages/osx/md5.md]: Maybe change the description to "...for a file" and remove the 1 in the token.

**Comment by igorshubovych** [comment]
Thank you @waldyrious. I updated PR.

**Pull request merged**